use_cases/init-process-for-fluent-bit/README.md

```diff
@@ -0,0 +1,227 @@
+# Init process for Fluent Bit on ECS, multi-config support
+
+
+
+### What is init process and multi-config support?
+
+Multi-config support means when you use a Fluent Bit image with `init` tag on ECS, you can use some config files conveniently to configure Fluent Bit. These config files can be your own files that you upload to S3, or you can use the [built-in config files](https://github.com/aws/aws-for-fluent-bit/tree/mainline/ecs) we provide directly. Init process achieves this feature. 
```

Comment:
> with `init` tag on ECS

A lot of our users are often very confused unless we are extremely careful... so let's give a sentence with the exact tags for some version to make it clear.

Reply:
Also nit/suggestion: "you can use some config files" change "some config files" => multiple config files or multiple config and parser files.

Reply:
Sure, will modify this